projects/1.20/assets/enchantment-descriptions/enchdesc/lang/zh_cn.json

```
@@ -0,0 +1,189 @@
+  "enchantment.create_sa.digging.desc": "使钻头可以钻出更大的洞。",
+  "enchantment.create_sa.hellfire.desc": "提高火焰喷射器的伤害，并使其能生成火。",
+  "__support_improved_exp": "https://www.curseforge.com/minecraft/mc-mods/improved-exp",
+  "enchantment.improved_exp.detection.desc": "攻击会给予目标发光效果。",
+  "enchantment.improved_exp.autosmelt.desc": "破坏方块时获得烧炼产物。",
+  "enchantment.improved_exp.pain_relief.desc": "按住Shift会消耗经验值以恢复生命值。",
+  "enchantment.improved_exp.heavyweight.desc": "加快穿戴者的摔落速度。",
+  "enchantment.improved_exp.saturation.desc": "恢复饥饿值和饱和度。",
+  "enchantment.improved_exp.boer_1.desc": "一次破坏多个方块。",
+  "enchantment.improved_exp.weight.desc": "降低使用者的移动速度。",
+  "enchantment.improved_exp.woodcutter.desc": "提升破坏木制方块的速度。",
+  "enchantment.improved_exp.coldblooded.desc": "当穿戴者在温暖生物群系时，会受到伤害。",
+  "enchantment.improved_exp.flammable.desc": "当穿戴者受到阳光直射时，会受到伤害。",
+  "enchantment.improved_exp.the_hatred_of_others.desc": "使敌对生物无视穿戴者。",
+  "enchantment.improved_exp.drill.desc": "显著提升挖掘方块的速度。",
+  "enchantment.improved_exp.habitual_place.desc": "根据所在维度，给予穿戴者相应的状态效果。",
+  "enchantment.improved_exp.medication.desc": "增加使用者的最大生命值。",
+  "enchantment.improved_exp.in_all_seriousness.desc": "当穿戴者生命值较低时，提升其移动速度。",
+  "enchantment.improved_exp.random_teleport.desc": "使用时朝随机方向传送。",
+  "enchantment.improved_exp.suspicious_look.desc": "使附近的生物注视着穿戴者",
+  "enchantment.improved_exp.resistance.desc": "给予穿戴者抗性提升效果。",
+  "enchantment.improved_exp.fear_of_depth.desc": "当穿戴者位于较深的水下时会受到伤害。",
+  "enchantment.improved_exp.jump_boost.desc": "提升穿戴者的跳跃力量。",
+  "enchantment.improved_exp.water_breathing.desc": "允许穿戴者在水下呼吸。",
+  "enchantment.improved_exp.bakingpowder.desc": "使锄一次转化多个方块",
+  "enchantment.improved_exp.damage_suppression.desc": "减少受到的伤害。",
+  "enchantment.improved_exp.auto_explosion.desc": "自动点燃放置的TNT。",
+  "enchantment.improved_exp.machine_gun.desc": "减少弓的蓄力时间。",
+  "enchantment.improved_exp.passage.desc": "使穿戴者飞向注视着的方块。",
+  "enchantment.improved_exp.preview.desc": "使穿戴者的视角向右旋转。",
+  "enchantment.improved_exp.evasion.desc": "可能使穿戴者回避伤害。",
+  "enchantment.improved_exp.explosive.desc": "当穿戴者生命值过低时会产生爆炸。",
+  "enchantment.improved_exp.night_vision.desc": "使穿戴者获得夜视效果。",
+  "enchantment.improved_exp.auto_repair.desc": "使工具的耐久度随时间不断恢复。",
+  "enchantment.improved_exp.poisonous_water.desc": "使穿戴者在水中中毒。",
+  "enchantment.improved_exp.harm.desc": "移除穿戴者身上全部的状态效果。",
+  "enchantment.improved_exp.pinger.desc": "耐久度过低时向穿戴者发出警告。",
+  "enchantment.improved_exp.warmblooded.desc": "当穿戴者在寒冷生物群系时，会受到伤害。",
+  "enchantment.improved_exp.slow_falling.desc": "减缓穿戴者的摔落速度。",
+  "enchantment.improved_exp.night.desc": "当穿戴者看不到天空时，会受到伤害。",
+  "enchantment.improved_exp.unstable.desc": "对使用者造成伤害。",
+  "enchantment.improved_exp.return_from_void.desc": "当穿戴者坠入虚空时，将其传送到附近的方块上。",
+  "enchantment.improved_exp.healing_waters.desc": "治疗处于水中的穿戴者。",
+  "enchantment.improved_exp.strength.desc": "提升穿戴者的攻击伤害。",
+  "enchantment.improved_exp.invulnerability.desc": "使穿戴者免疫伤害。",
+  "enchantment.improved_exp.true_shot.desc": "以更大的力量射箭。",
```

| **审阅级别** |
|:--------------:|
|⚪ **轻微** / 不阻止合并但需三思|

原文：**Arrows are fired with more power.**
```suggestion
  "enchantment.improved_exp.true_shot.desc": "射箭更有力。",
```